### ISSUE TITLE
Pytorch AvgPool转onnx再转到MNN，结果无法对齐

### ISSUE THREAD

**Issue by mna12478** [alibaba/MNN#817]
输入shape=[1, 3, 224, 224]
pytorch模型结构：oup=torch.nn.AvgPool2d(kernel_size=3, stride=2, padding=(1,1))
pytorch输出shape=[1, 3, 112, 112]

torch.onnx.export导出onnx模型

onnx：
%300：Tensor = onnx::Constant[value= 0 0 1 1 0 0 1 1 [CPULongType{8}]]()
%301：Tensor = onnx::Pad[mode="constant"](%input, %300)
%384：Float(1, 3, 112, 112) = onnx::AveragePool[ceil_mode=0, kernel_shape=[3,3], pads=[0,0,0,0], strides=[2, 2]](%301)
onnx输出shape=(1, 3, 112, 112)

MNN：输出shape=(1, 5, 112, 111)

**Comment by jxt1234** [comment]
相关的 onnx 模型发一下?

**Comment by mna12478** [comment]
> 相关的 onnx 模型发一下?

模型已邮件发送，请查收
